2.2/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/base-centos7
+
+# This image provides a Ruby 2.2 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
`MAINTAINER` is recommended, but for RHEL images we shouldn't use it actually, it would just cause issues in case the maintainer is not available any more. At least I was told better not to use it at all..